### ISSUE TITLE
hack/verify-godeps.sh is failing on release-1.1 branch

### ISSUE THREAD

**Issue by piosz** [kubernetes/kubernetes#15297]
```
package github.com/aws/aws-sdk-go/internal/apierr
	imports github.com/aws/aws-sdk-go/internal/apierr
	imports github.com/aws/aws-sdk-go/internal/apierr: cannot find package "github.com/aws/aws-sdk-go/internal/apierr" in any of:
	/root/.gvm/gos/go1.4/src/github.com/aws/aws-sdk-go/internal/apierr (from $GOROOT)
	/tmp/gopath.JUrP0w/src/github.com/aws/aws-sdk-go/internal/apierr (from $GOPATH)
godep: restore: exit status 1
package github.com/google/google-api-go-client/cloudmonitoring/v2beta2
	imports github.com/google/google-api-go-client/cloudmonitoring/v2beta2
	imports github.com/google/google-api-go-client/cloudmonitoring/v2beta2: code in directory /tmp/gopath.JUrP0w/src/github.com/google/google-api-go-client/cloudmonitoring/v2beta2 expects import "google.golang.org/api/cloudmonitoring/v2beta2"
godep: restore: exit status 1
!!! Error in ./hack/verify-godeps.sh:69
  '"${GODEP}" restore' exited with status 1
Call stack:
  1: ./hack/verify-godeps.sh:69 main(...)
Exiting with status 1
```

**Comment by piosz** [comment]
Shippable failed after
#15217 was merged https://app.shippable.com/builds/56163362bf5f9b0c00c4a7e4
#15155 was merged https://app.shippable.com/builds/561600acbf5f9b0c00b406ac

**Comment by piosz** [comment]
fixed by #15298
